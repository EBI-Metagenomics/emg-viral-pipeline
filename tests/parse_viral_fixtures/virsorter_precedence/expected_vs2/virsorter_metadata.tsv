contig	category	circular	prophage_start	prophage_end
seq1	high_confidence	False
seq2	high_confidence	False
seq4	high_confidence	False
seq3	high_confidence	False
seq5	high_confidence	False
seq6	high_confidence	False
seq9	high_confidence	False
seq7	low_confidence	True
seq8	low_confidence	False
seq10	low_confidence	False
seq1	prophage	False	1	23146
