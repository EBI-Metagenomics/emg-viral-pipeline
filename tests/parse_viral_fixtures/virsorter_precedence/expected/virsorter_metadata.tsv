contig	category	circular	prophage_start	prophage_end
seq6	high_confidence	False
seq7	high_confidence	False
seq8	high_confidence	False
seq9	high_confidence	False
seq2	high_confidence	False
seq3	high_confidence	False
seq1	prophage	False	21696	135184
